MAIN_minimalistic.m

```diff
@@ -0,0 +1,40 @@
+function MAIN_minimalistic()
+close all; clear; clc;
+
```

Comment:
Add a sentence or two here describing the physical situation what you are simulating.  Quickly looking through the code I think it is a one-dimensional bouncing ball.